README.md

```diff
@@ -1,12 +1,14 @@
-# Calculator
+# Calculator ![Calculator Icon](docs/Images/CalculatorIcon.png)
 The Windows Calculator app is a modern Windows app written in C++ that ships pre-installed with Windows.
 The app provides standard, scientific, and programmer calculator functionality, as well as a set of converters between various units of measurement and currencies.
 
 Calculator ships regularly with new features and bug fixes. You can get the latest version of Calculator in the [Microsoft Store](https://www.microsoft.com/store/apps/9WZDNCRFHVN5).
 
 [![Build Status](https://dev.azure.com/ms/calculator/_apis/build/status/Calculator-CI?branchName=master)](https://dev.azure.com/ms/calculator/_build/latest?definitionId=57&branchName=master)
 
-  ![Calculator Screenshot](docs/Images/CalculatorScreenshot.png)
+<p align="center">
```

Comment:
I'm a bit more mixed on this change.  I would totally be down for centering the screenshot if Markdown natively supported doing so, but it doesn't.  I'm not sure if we're gaining _sufficient_ "awesomeness" in this instance by veering away from native Markdown, so I'd probably just revert this part of the change.